Core/include/Acts/Vertexing/VertexingOptions.hpp

```diff
@@ -37,6 +37,7 @@ struct VertexingOptions {
   /// Context object for the geometry
   std::reference_wrapper<const GeometryContext> geoContext;
   /// Context object for the magnetic field
+  // @TODO: Can this go?
```

Comment:
What about this?

Reply:
I wasn't sure if the mag field context was needed in this struct anymore.